CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ set(CMAKE_INSTALL_MESSAGE NEVER)
 option(MLX_BUILD_TESTS "Build tests for mlx" ON)
 option(MLX_BUILD_EXAMPLES "Build examples for mlx" ON)
 option(MLX_BUILD_BENCHMARKS "Build benchmarks for mlx" OFF)
-option(MLX_BUILD_PYTHON_BINDINGS "Build python bindings for mlx" OFF)
```

Comment:
Could you revert this? Not sure what the intention is.. but we don't build them by default intentionally

Reply:
I struggled a bit with cmake at first, so I was playing the settings to understand what to do. I forgot to revert before committing